posts/Davide/2017-10-16-hibernate-ogm-5-2-Beta1-released.adoc

```diff
@@ -0,0 +1,78 @@
+= "Hibernate OGM 5.2 Beta1 is out"
+Davide D'Alto
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+Another month, another release: http://hibernate.org/ogm/[Hibernate OGM] 5.2 Beta1 is ready!
+
+The main goal for this release is to support Infinispan 9.1 and allow the creations of caches
+when they are missing.
+
+If you need to upgrade from a version before 5.2, you can find help on the
+https://developer.jboss.org/wiki/HibernateOGMMigrationNotes[migration notes].
```

Comment:
Not sure it's worth mentioning, since the 5.2 migration notes only state "In progress"...

Reply:
Good catch, I will update the migration guide